config/manifests/kustomization.yaml

```diff
@@ -6,3 +6,19 @@ resources:
 - ../scorecard
 apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I think we should remove the CSV file and its reference there? @anishasthana @sutaakar can you confirm?

Reply:
I think so

Reply:
The CSV file?

Reply:
The `bases/codeflare-operator.clusterserviceversion.yaml` file.

Reply:
Ah so I forgot we still run the OLM upgrade test. @sutaakar what's your opinion on this? Do we want to keep them around, in which case we need to keep the CSV?